flink/interpreter/src/main/java/org/apache/zeppelin/flink/JobManager.java

```diff
@@ -218,15 +222,39 @@ public void run() {
       }
     }
 
-      public void cancel () {
-        this.running.set(false);
-        synchronized (running) {
-          running.notify();
-        }
+    public void cancel() {
+      this.running.set(false);
+      synchronized (running) {
+        running.notify();
       }
+    }
 
-      public int getProgress () {
-        return progress;
-      }
+    public int getProgress() {
+      return progress;
     }
   }
+
+  static String toRichTimeDuration(long duration) {
```

Comment:
would be useful to add a javadoc for it, saying what is parameter is - seconds, milliseconds, etc.?

Reply:
javadoc is added